script.js

```diff
@@ -1,11 +1,17 @@
 // Create an array called `colors` that contains three colors.
+var colors = ["red", "green", "blue"];
 
 // Log the last item in the array to the console.
-
+colors [2]
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
 
 // Create an `instructor` object that contains three key-value pairs.
+var instructor = {name: Adrian, hair color: brown, computer;mac};
```

Comment:
A couple of notes...

* While an object key can contain spaces, as you do with "hair color", this is not how you go about instantiating it. You would have to wrap the key in quotes.
* A key must be followed by a colon. You used a semi-colon with your `computer` key.
* Definitely test your answers before committing. You would've seen the errors for this one.